src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/DelegationsController.cs

```diff
@@ -72,7 +72,7 @@ public async Task<ActionResult> Post([FromBody] List<Rule> rules)
                 }
 
                 string rulesJson = JsonSerializer.Serialize(rules);
-                _logger.LogError($"Delegation could not be completed. None of the rules could be processed, indicating invalid or incomplete input:\n{rulesJson}", rulesJson);
+                _logger.LogError("Delegation could not be completed. None of the rules could be processed, indicating invalid or incomplete input:\n{rulesJson}", rulesJson);
                 return StatusCode(400, $"Delegation could not be completed");
```

Comment:
This is the way I see it inconsistent use of severity. An error occurs because of faults in the application, status 400 is given if there is something wrong with the input from the client.

I want us to consider the option of not logging here at all. Is this information valuable to us? In this case we also control the client. Maybe this is something the client can log and handle as error instead?

Why aren't we returning delegation results in this case? Are there no information about why a rule was skipped?

The same questions apply to the other cases.

Reply:
It would require some changes to the return model to be able to hold information on what was causing deletion to be less than expected. Must discuss this with team Authorization what to do here.

Reply:
Agree that this probably isn't an error. I think LogError should only be used when we're fairly sure there is an actual bug/state desync. In this particular case, this might occur due to multi-tab concurrency issues, and is thus not entirely unexpected, which the severity and error message should reflect. Blatant client errors, ie the wrong/empty model, are handled earlier in the method (without explicit logging, which is OK because we don't really have any additional information to log). 

I think however that right here a LogWarning is warranted. At this point we have additional information (the input rules) that might prove useful in assessing if this is an actual bug or not. If this gets noisy, we can always reduce the severity or remove it alltogether later. Generous logging early in the lifespan of this component lets us know more about how it is used and how it behaves, and we will most likely have to tune what we do and do not log later anyway as its use begins to pick up.

In general, I believe logging should primarily be done in Altinn3, where we usually have all the information available in order to log something actionable. The tools we have for logging/monitoring, and how accessible they are for VU, is also vastly superior in Altinn3 compared to Altinn2. Hence, the information returned to Altinn2 doesn't have to exceed what is needed in order to show the end-user that something went ok, partially ok or failed. Exactly why either of those things happened isn't particulary useful to the end-user to know in my opinion.